model/things/src/main/java/org/eclipse/ditto/model/things/Thing.java

```diff
@@ -653,6 +653,13 @@ default void validate(DittoHeaders headers) { }
                 JsonFactory.newJsonObjectFieldDefinition("features", FieldType.REGULAR, JsonSchemaVersion.V_1,
                         JsonSchemaVersion.V_2);
 
+        /**
+         * JSON field containing the Thing's metadata.
+         */
+        public static final JsonFieldDefinition<JsonObject> METADATA =
+            JsonFactory.newJsonObjectFieldDefinition("_metadata", FieldType.SPECIAL, FieldType.HIDDEN,
+                JsonSchemaVersion.V_1, JsonSchemaVersion.V_2);
+
```

Comment:
I would move this up below the "_modified" definition to have them all in one place.